tests/test_helpers.py

```diff
@@ -442,6 +442,44 @@ def test_content_disposition() -> None:
         helpers.content_disposition_header("attachment", foo="bar")
         == 'attachment; foo="bar"'
     )
+    assert (
+        helpers.content_disposition_header("attachment", foo="bar[]")
+        == 'attachment; foo="bar[]"'
+    )
+    assert (
+        helpers.content_disposition_header("attachment", foo=' a""b\\')
+        == 'attachment; foo="\\ a\\"\\"b\\\\"'
+    )
+    assert (
+        helpers.content_disposition_header("attachment", foo="bär")
+        == "attachment; foo*=utf-8''b%C3%A4r"
+    )
+    assert (
+        helpers.content_disposition_header(
+            "attachment", foo='bär "\\', quote_fields=False
+        )
+        == 'attachment; foo="bär \\"\\\\"'
+    )
+    assert (
+        helpers.content_disposition_header("attachment", foo="bär", _charset="latin-1")
+        == "attachment; foo*=latin-1''b%E4r"
+    )
+    assert (
+        helpers.content_disposition_header("attachment", filename="bär")
+        == 'attachment; filename="b%C3%A4r"'
+    )
+    assert (
+        helpers.content_disposition_header(
+            "attachment", filename="bär", _charset="latin-1"
+        )
+        == 'attachment; filename="b%E4r"'
+    )
+    assert (
+        helpers.content_disposition_header(
+            "attachment", filename='bär "\\', quote_fields=False
+        )
+        == 'attachment; filename="bär \\"\\\\"'
+    )
```

Comment:
It's better to have this use `@pytest.mark.parametrize` because it's many tests, not one.

Reply:
The minor thing I believe. 
If @kohtala wants to update tests -- that's perfect.
If no -- I think we can accept tests as is.

P.S.
Our code is not ideal but just 'good enough' :)

Reply:
Ok. It looked nicer too.

Reply:
I parameterized the tests yesterday, but was in a hurry and failed to push. I pushed it now. Thanks for the idea. It also looks cleaner this way.